schema/field.go

```diff
@@ -56,6 +56,7 @@ type Field struct {
 	BindNames              []string
 	DataType               DataType
 	GORMDataType           DataType
+	CustomDataType         DataType
```

Comment:
The meaning of the name CustomDataType is a bit vague to me.
It seems that we only need to know whether the user has specified the tag type, but not what the type is, because it will be equal to the datatype.

Reply:
Do you have any good ideas about naming? 
How about "IsTaggedType"?

Reply:
I don't have a better suggestion, but IsTaggedType would work better for me than CustomDataType